crypto/Cargo.toml

```diff
@@ -35,3 +36,5 @@ rand = "0.8"
 chacha20poly1305 = "0.9.0"
 # only needed because ark-ff doesn't display correctly
 num-bigint = "0.4"
+# this allows us to implement Decode for crypto types without orphan rule issues
+sqlx = { version = "0.5.9", features = ["postgres"], optional = true }
```

Comment:
This is a bit awkward but seems like the least bad way to have sqlx impls for the crypto types that allow us to store them in the DB directly without having to write an extra translation layer.